architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management for Kubernetes, or as a standalone operator that is supported as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription.
+
+If you use Red Hat Advanced Cluster Management, multicluster engine is included and provides the following capabilities:
+
+* klusterlet agent, which registers managed clusters to the hub.
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes Central Infrastructure Management (CIM), which provides an interactive user interface in the Red Hat Advanced Cluster Management console where you can provision clusters.
+* Open cluster management, which provides resources to manage multiple Kubernetes clusters.
+
+If you use {product-title} 4.8.2 or later, you can separately install multicluster engine to access the following capabilities:
+
+* Hive, which provisions and manages self-managed OpenShift clusters to the hub.
```

Comment:
Suggest to expand to the following:
Hive, which provisions and performs initial configurations for self-managed OpenShift clusters to the hub.